src/manual/using/integration/frontend.md

```diff
@@ -32,9 +32,292 @@ Taskcluster API and refresh when necessary.
 
 The
 [taskcluster-oidc-test](https://github.com/taskcluster/taskcluster-oidc-test)
-repository provides an example of a sipmle SPA following this process.  You can
+repository provides an example of a simple SPA following this process.  You can
 see it in action at https://taskcluster-oidc-test.herokuapp.com.
 
+## Creating a simple login integration
+
+Authorizing your application to use Taskcluster APIs from a frontend project is relatively
+straightforward, but there can be some hurdles to jump through. First, let's talk about a
+few prerequisites:
+
+* Possess an [auth0 Client](https://auth0.com/docs/clients) which can access Taskcluster
+credentials. If you don't have one, you can request a [New Single Sign On Application in
+The Hub](https://mozilla.service-now.com/sp?id=sc_cat_item&sys_id=1e9746c20f76aa0087591d2be1050ecb).
+Since this is going to be used on the frontend, make sure to request a client that uses
+"RS256 algorithms" and "OpenIDConnect (OIDC)".
```

Comment:
Also mention this should be a single-page application, and that RS256 and single-page need to be specified in the comments at the bottom of the form.